api/document/templates/document/editor_akn.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <style>
+    #editor {
+      border: 1px solid black;
+    }
+
+    .error {
+        color: red;
+    }
+
+    /* https://meta.superuser.com/q/4788 */
+    kbd {
+      -moz-border-radius:3px;
+      -moz-box-shadow:0 1px 0 rgba(0,0,0,0.2),0 0 0 2px #fff inset;
+      -webkit-border-radius:3px;
+      -webkit-box-shadow:0 1px 0 rgba(0,0,0,0.2),0 0 0 2px #fff inset;
+      background-color:#f7f7f7;
+      border:1px solid #ccc;
+      border-radius:3px;
+      box-shadow:0 1px 0 rgba(0,0,0,0.2),0 0 0 2px #fff inset;
+      color:#333;
+      display:inline-block;
+      font-family:Arial,Helvetica,sans-serif;
+      font-size:11px;
+      line-height:1.4;
+      margin:0 .1em;
+      padding:.1em .6em;
+      text-shadow:0 1px 0 #fff;
+    }
+    </style>
+    <title>Akoma Ntoso Document editor</title>
+  </head>
+  <body>
+    <h1>Akoma Ntoso Document editor</h1>
+    <pre id="status"></pre>
+    <div id="editor" data-document-url="{{ document_url }}"></div>
```

Comment:
I like passing in the whole url much more than just the doc number. Do we need to do anything around escaping here? It might be a good idea to use the `{% url %}` directive to handle that for us.

Not for now, but I wonder if there's a meta-data or other RDFa tag that'd make this semantic.

Reply:
Hmm, I don't think that we need to do any escaping there--I had decided to just pass it in from the view function so that the view functions for our prosemirror + codemirror editors could reuse the same logic for coming up w/ that URL, rather than dupicating it in the templates, but I guess either way works!